src/DynamoCoreWpf/Commands/WorkspaceCommands.cs

```diff
@@ -11,11 +17,20 @@ public partial class WorkspaceViewModel
         private DelegateCommand _resetFitViewToggleCommand;
         private DelegateCommand _findByIdCommand;
         private DelegateCommand _alignSelectedCommand;
+        private DelegateCommand _enableNodePreviewCommand;
+        private DelegateCommand _enableUpstreamPreviewCommand;
+        private DelegateCommand _setArgumentLacingCommand;
```

Comment:
Hi @nguyen-binh-minh, I don't think the above two `_enableXyzCommand` are used, can you remove them?